content/reference/user/private_locations/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Private Locations"
+description: "Host Load Generators on your private AWS account"
+lead: "Private Locations on your AWS account"
+date: 2021-11-07T14:29:04+00:00
+lastmod: 2021-11-07T14:29:04+00:00
+weight: 21070
+---
+
+A location describe where to start load generators:
+- Public locations are available by default, and listed by regions.
+- Private locations must be configured, and allow you to deploy load generators in your infrastructure.
+
+{{< alert tip >}}
+Private locations allow you to load test in a private infrastructure, without any inbound connection or credential sharing.
+To access this feature, please contact our [technical support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8/group/12/create/59?summary=Private+Locations&description=Contact%20email%3A%20%3Cemail%3E%0A%0AHello%2C%20we%20would%20like%20to%20enable%20the%20private%20locations%20feature%20on%20our%20organization.).
+{{< /alert >}}
+
+## Introduction
+
+To configure private location, you must configure an agent in your infrastructure, called a control plane.
```

Comment:
```suggestion
To configure private locations, you must configure an agent in your infrastructure, called a control plane.
```
(or "a private  location", but they can configure several if they want, so I'd use the plural)